blocks/log.py

```diff
@@ -245,3 +254,18 @@ def __iter__(self):
 
     def get_status(self):
         return self._status
+
+    def to_dataframe(self):
+        return DataFrame.from_dict(self._storage, orient='index')
+
+
+def to_dataframe(log):
+    """Convert a log into a :class:`.DataFrame`."""
+    if not pandas_available:
+        raise Exception("The pandas library is not found. You can"
```

Comment:
Just raise an `ImportError` here. (In general, I don't think there are many cases where you want to raise `Exception`, one of its more specific subclasses is almost always better.)